android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/SplashScreenActivity.java

```diff
@@ -0,0 +1,48 @@
+package com.tavultesoft.kmapro;
+
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.os.Handler;
+import android.app.Activity;
+import android.content.Intent;
+import android.widget.TextView;
+
+import java.util.Calendar;
+
+public class SplashScreenActivity extends Activity {
+
+  private static int SPLASH_TIME_OUT = 1000; // msec
```

Comment:
I know it's only a second, but I hate adding an artificial delay into the startup of the app, especially to display only version+copyright data. Perhaps we just leave that extra data out for now? -- leave the code in but stub out the showing of it?)

Reply:
k, I've set the timeout to `0` so the text portion of the splash screen will immediately timeout to the MainActivity.